src/flatc.cpp

```diff
@@ -115,6 +115,7 @@ std::string FlatCompiler::GetUsageString(const char *program_name) const {
     "  --no-ts-reexport   Don't re-export imported dependencies for TypeScript.\n"
     "  --reflect-types    Add minimal type reflection to code generation.\n"
     "  --reflect-names    Add minimal type/name reflection.\n"
+    "  --prefix-size      Prefix size field to generated buffers.\n"
```

Comment:
Maybe note that this only applies to parsed JSON?

Reply:
it is applicable for parsed JSON and schema serialization, right?

Reply:
True.. just thinking of a way this can be clearer. Maybe `wire format (-b)` instead of `buffers` ?

Reply:
That makes sense, I'll add that.